Formula/ddgr.rb

```diff
@@ -3,10 +3,9 @@ class Ddgr < Formula
 
   desc "DuckDuckGo from the terminal"
   homepage "https://github.com/jarun/ddgr"
-  url "https://github.com/jarun/ddgr/archive/v1.9.tar.gz"
-  sha256 "3dfe82fab649f1cec904a1de63f78692be329a3b6928c1615f22c76f6e21c36f"
+  url "https://github.com/jarun/ddgr/archive/v2.0.tar.gz"
+  sha256 "7e46430b0a8c479a5feca832adb73f2f09804bf603dedc50f4cf2e1da4c75f88"
   license "GPL-3.0"
```

Comment:
Code header contains (see for example https://github.com/jarun/ddgr/blob/main/ddgr)
```
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
```

```suggestion
  license "GPL-3.0-or-later"
```